stdlib/public/stubs/Stubs.cpp

```diff
@@ -33,6 +33,20 @@
 #include <sstream>
 #include <cmath>
 #define fmodl(lhs, rhs) std::fmod(lhs, rhs)
+#elif defined(__ANDROID__)
+#include <locale.h>
+static double swift_strtod_l(const char *nptr, char **endptr, locale_t loc) {
```

Comment:
Please add a comment that this is only valid on Android because it does not support locales other than C.  (Is that actually true?)